projects/core/src/features-config/directives/feature-styles.service.ts

```diff
@@ -0,0 +1,163 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ComponentRef,
+  Injectable,
+  RendererFactory2,
+  inject,
+  isDevMode,
+} from '@angular/core';
+import { LoggerService } from '../../logger/logger.service';
+import { FeatureConfigService } from '../services/feature-config.service';
+
+/**
+ * Automatically adds/removes CSS classes to/from the root element of the application,
+ * based on the explicitly registered usage of feature flags.
+ *
+ * The CSS classes are added only for flags that are currently used and enabled in the config.
+ *
+ * The CSS classes for flags that are enabled BUT NOT USED are NOT ADDED to the root element.
+ */
+@Injectable({ providedIn: 'root' })
+export class FeatureStylesService {
```

Comment:
Shouldn't we put this to parallel folder `projects/core/src/features-config/services `?

Reply:
Thanks. I also noticed now it's not exported in public API, not deliberately

Reply:
moved to /services